styles.css

```diff
@@ -1,77 +1,98 @@
+body {
+  background-color: #172b4d;
+}
+
 .container {
-    border: 5px solid #344563;
-    width: 300px;
-    height: 100vh;
-    margin: 0 auto;
-    padding: 20px;
-    background-image: url(/Graphics/header-shapes\ mobile.png);
-    background-size: contain;
+  margin: 0 auto;
+  width: 315px;
+  height: 710px;
+  background-color: #fff;
+  background-image: url(./Graphics/header-shapes\ mobile.png); /* fixed the path */
+  background-size: contain;
+  background-position: center;
+  padding: 30px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] Background image should be the same as the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=2mud6GRzeeuM1rW3-0) to look your project professional. Kindly fix this issue.

***Your Design***
![Screenshot from 2023-01-12 02-10-45](https://user-images.githubusercontent.com/60083437/211908294-b5c1f31e-70bb-4ca4-a365-51f077b26758.png)

***Figma Design***
![Screenshot from 2023-01-12 02-13-49](https://user-images.githubusercontent.com/60083437/211908397-105a89de-0a68-4754-961b-1987e311945f.png)